Products/PortalTransforms/transforms/safe_html.py

```diff
@@ -2574,8 +2574,8 @@ def strip_outer(s):
                           embedded=False,
                           remove_unknown_tags=True,
                           meta=False,
-                          javascript=bool('script' in self.config['nasty_tags']),
-                          scripts=bool('script' in self.config['nasty_tags']),
+                          javascript=('script' in self.config['nasty_tags']),
+                          scripts=('script' in self.config['nasty_tags']),
```

Comment:
I don't want to invest more time in this. If anyone wants to change it, go ahead.